dockerfiles/che/Dockerfile.centos

```diff
@@ -46,7 +46,7 @@ EXPOSE 8000 8080
 
 USER user
 
-ADD assembly/assembly-main/target/eclipse-che-*/eclipse-che-* /home/user/che/
+ADD assembly /home/user/che
 
 ENTRYPOINT [ "/home/user/che/bin/che.sh", "-c" ]
```

Comment:
why should I update the entrypoint ? the current dockerfile is not using docker.sh script (which is now entrypoint.sh) so maybe there were some reasons ?